application/wypl-core/src/main/java/com/wypl/wyplcore/facade/AuthMemberFacadeImpl.java

```diff
@@ -0,0 +1,78 @@
+package com.wypl.wyplcore.facade;
+
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.wypl.googleoauthclient.GoogleOAuthClient;
+import com.wypl.googleoauthclient.data.response.GoogleTokenResponse;
+import com.wypl.googleoauthclient.data.response.GoogleUserInfoResponse;
+import com.wypl.googleoauthclient.domain.AuthMember;
+import com.wypl.googleoauthclient.exception.GoogleOAuthErrorCode;
+import com.wypl.googleoauthclient.exception.GoogleOAuthException;
+import com.wypl.wyplcore.auth.data.response.AuthTokensResponse;
+import com.wypl.wyplcore.member.service.MemberService;
+import com.wypl.wyplcore.token.service.TokenService;
+
+import lombok.RequiredArgsConstructor;
+
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+@Component
+public class AuthMemberFacadeImpl implements AuthMemberFacade {
```

Comment:
```suggestion
public class AuthMemberFacadeService implements AuthMemberService {
```

어떠한 기능을 하는 서비스가 있는데 해당 서비스를 구성한 디자인패턴과 같은.. 내용을 클래스에 붙힘